.github/workflows/build.yml

```diff
@@ -54,15 +54,18 @@ jobs:
         env:
           ARCH: ${{ matrix.arch }}-${{ env.BRANCH }}
           FEEDNAME: packages_ci
+          V: s
 
       - name: Move created packages to project dir
-        run: cp bin/packages/${{ matrix.arch }}/packages_ci/*.ipk . || true
+        run: cp bin/packages/${{ matrix.arch }}/packages_ci/* . || true
```

Comment:
I wonder if this should be `*.?pk` instead ?  
To match both `.apk` and `.ipk`, but not removing the selection criteria totally leaving just the `*` wildcard there.

Reply:
Should we bite the bulled and be more explicit?
```
cp bin/packages/${{ matrix.arch }}/packages_ci/*.ipk . || true
cp bin/packages/${{ matrix.arch }}/packages_ci/*.apk . || true
```

Reply:
Prefer one-liners :D

Reply:
I'm fine with either solution, I don't have any real preference as long as we can capture the .apk files.

Reply:
I saw that actions-shared-workflows uses the [same principle](https://github.com/openwrt/actions-shared-workflows/blob/main/.github/workflows/multi-arch-test-build.yml#L119) and then [copies the relevant files from the working directory](https://github.com/openwrt/actions-shared-workflows/blob/main/.github/workflows/multi-arch-test-build.yml#L165-L171) so maybe we should keep it the way it is now?

Reply:
Well if we want to keep consistency yes, we should use the * directly.